src/lib.rs

```diff
@@ -262,19 +291,62 @@ impl Scope {
             .or_insert_with(|| Import::new(path, ty))
     }
 
+    /// Returns true if a module named `name` exists in this scope.
+    pub fn contains_module<Q: ?Sized>(&self, name: &Q) -> bool 
```

Comment:
We probably don't need this function right now. I would stick with `get_module` and `get_module_mut`. These functions return `Option` so you can just do `if scope.get_module("name").is_some()`.

Again, we can re-evaluate later if it ends up sucking.